.mailmap

```diff
@@ -0,0 +1,27 @@
+Andreas Haas <liu.gam3@gmail.com>
+Andrew Conrad <aconrad103@gmail.com>
+Andrii Doroshenko <xrayez@gmail.com>
+puchik <puchik@users.noreply.github.com> <48544263+puchik@users.noreply.github.com>
+Chris Bradfield <chris@kidscancode.org> <cb@scribe.net>
+clayjohn <claynjohn@gmail.com>
+clayjohn <claynjohn@gmail.com> <clayjohn@shaw.ca>
+corrigentia <coryan25@gmail.com> <20541985+corrigentia@users.noreply.github.com>
+Frido <f.schermutzki@gmail.com>
+Hugo Locurcio <hugo.locurcio@hugo.pro> <calinou@opmbx.org>
+Hugo Locurcio <hugo.locurcio@hugo.pro> <hugo.l@openmailbox.org>
+Ignacio Etcheverry <ignalfonsore@gmail.com> <neikeq@users.noreply.github.com>
+Julian Murgia <the.straton@gmail.com>
+Kelly Thomas <kelly.thomas@hotmail.com.au>
+Leon Krause <lk@leonkrause.com>
+Leon Krause <lk@leonkrause.com> <eska@eska.me>
+Max Hilbrunner <m.hilbrunner@gmail.com>
+Max Hilbrunner <m.hilbrunner@gmail.com> <mhilbrunner@users.noreply.github.com>pn
```

Comment:
```suggestion
Max Hilbrunner <m.hilbrunner@gmail.com> <mhilbrunner@users.noreply.github.com>
```

Is this a typo? Couldn't find anything that documents a meaning for `pn` in `.mailmap`, and my entry in the main repo lacks this :)

Reply:
That was to check if you do your PR reviews thoroughly ;P

Reply:
Nah, we both know I only check *yours* thoroughly so I can nitpick endlessly :P